crates/integrations/cli/testdata/catalogs.toml

```diff
@@ -0,0 +1,37 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[[catalogs]]
+name = "demo"
+type = "rest"
+uri = "http://localhost:8080"
+warehouse = "s3://iceberg-demo"
+
+[catalogs.props]
+"s3.endpoint" = "http://localhost:9000"
+"s3.access_key_id" = "admin"
+
+
+[[catalogs]]
+name = "demo2"
+type = "rest"
+uri = "http://localhost2:8080"
+warehouse = "s3://iceberg-demo2"
+
+[catalogs.props]
+"s3.endpoint" = "http://localhost2:9090"
+"s3.access_key_id" = "admin2"
```

Comment:
Can we use a format like this:
i.e., put `name` in the table, instead using Array of tables. I think this can be more user-friendly.

The main reason is that `[catalogs.props]` is not very clear which catalog does it belong to.

```toml
[catalogs.demo]
type = "rest"
uri = "http://localhost:8080/"
warehouse = "s3://iceberg-demo"

[catalogs.demo.props]
"s3.endpoint" = "http://localhost:9000/"
"s3.access_key_id" = "admin"


[catalogs.demo2]
type = "rest"
uri = "http://localhost2:8080/"
warehouse = "s3://iceberg-demo2"

[catalogs.demo2.props]
"s3.endpoint" = "http://localhost2:9090/"
"s3.access_key_id" = "admin2"
```